doc/introduction/index.rst

```diff
@@ -87,6 +87,64 @@ platform to run both safety-critical applications and non-safety
 applications, together with security functions that safeguard the
 system.
 
+Industrial Workload Consolidation
+=================================
+
+.. figure:: images/ACRN-V2-industrial-scenario.png
+   :width: 600px
+   :align: center
+   :name: V2-industrial-scenario
+
+   ACRN Industrial Workload Consolidation scenario
+
+Supporting Workload consolidation for industrial applications is even
+more challenging. The ACRN hypervisor needs to run different workloads with no
+interference, increase security functions that safeguard the system, run hard
+real-time sensitive workloads together with general computing workloads, and
+conduct data analytics for timely actions and predictive maintenance.
+
+Virtualization is especially important in industrial environments
+because of device and application longevity. Virtualization enables
+factories to modernize their control system hardware by using VMs to run
+older control systems and operating systems far beyond their intended
+retirement dates.
+
+As shown in :numref:`V2-industrial-scenario`, the Service VM can start a number
+of post-launched User VMs and can provide device sharing capabilities to these.
+In total, up to 7 post-launched User VMs can be started:
```

Comment:
The User VM num limitation is only for CPU sharing feature is disabled (i.e. CONFIG_SCHED_NOOP=y). The max VM num is limited by logic core num in this case.
But when CPU sharing is enabled, we have no such limitation, ACRN could support a large VM number if memory is enough. To avoid complexity, currently we set the VM num limit to 8.

Reply:
@dbkinder I have not included this in the latest version. CPU sharing is, by default, enabled so the limit is somewhat arbitrary and can be modified by modified the scenario files.